r2i/tflite/engine.cc

```diff
@@ -130,6 +146,12 @@ std::shared_ptr<r2i::IPrediction> Engine::Predict (std::shared_ptr<r2i::IFrame>
     return nullptr;
   }
 
+  if (this->number_of_threads != -1) {
```

Comment:
We should always set it, even if it is `-1`. Imagine this:
1. SetNumThreads(4)
2. Predict
3. SetNumThreads(-1)
4. Predict <-----This continues to use 4 threads.